api/src/main/java/net/kyori/adventure/text/format/TextColor.java

```diff
@@ -63,10 +64,46 @@
    * @return a new text colour
    * @since 4.0.0
    */
-  static @NonNull TextColor color(final RGBLike rgb) {
+  static @NonNull TextColor color(final @NonNull RGBLike rgb) {
     return color(rgb.red(), rgb.green(), rgb.blue());
   }
 
+  /**
+   * Creates a new text color.
```

Comment:
A bit more descriptive here please (maybe link to wiki article about HSV, etc) - chances are people will be far less familiar with HSV compared to RGB.

Reply:
I've added a little bit more description, and linked to a wiki article like you mentioned. Let me know if you have any further suggestions to improve the doc.